assets/css/rating-amp.css

```diff
@@ -128,4 +128,14 @@
     -webkit-transform: rotate(36deg);
     -ms-transform: rotate(36deg);
     transform: rotate(36deg);
+}
+
+.wppr-inline-pie-chart {
+    position: relative;
```

Comment:
why those are needed ?

Reply:
@selul In [this](https://demo.themeisle.com/wp-product-review/shortcode-review/amp/) page, you'll see a circle rating floating and it is not on it's actual position. It is because these properties were missing. Just added them in this class.

Reply:
got it.